### DISCUSSION TITLE
`divisor` in Attributes

### DISCUSSION THREAD

**Discussion by ilan-gold** [visgl/deck.gl#6128] [Q&A] [answered]
I was recently using custom attributes with a `PolygonLayer` and I noticed that `divisor` [was `undefined` and not 0](https://github.com/vitessce/vitessce/pull/1024/commits/7d4b83118e92ef2aa3b2bec9a1eacb89c6ced025) even though I had not set it.  The attribute was not working, but once I set it to 0, it worked.  It seems like this line https://github.com/visgl/deck.gl/blob/437fc796fa60f555536954edbe725a09731d6f1f/modules/core/src/lib/attribute/attribute-manager.js#L307 could return `undefined`.  I only ask because it seems like [the docs](https://deck.gl/docs/api-reference/core/attribute-manager#add) say the default is 0.  Should the docs be updated?  Am I missing something.

**Comment by Pessimistress** [comment, marked as answer]
Can you open a PR?

The default value should be enforced by https://github.com/visgl/luma.gl/blob/master/modules/webgl/src/classes/accessor.js, though it doesn't seem so... @ibgreen

**Reply by ilan-gold** [reply]
Sure @Pessimistress.
